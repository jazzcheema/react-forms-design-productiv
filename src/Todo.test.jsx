import { describe, it, expect } from "vitest";

import React from "react";
import { render, fireEvent } from "@testing-library/react";
import Todo from "./Todo";

describe("Todo component", function () {

    const testTodo = {
        title: "Test Title1",
        description: "Test Description1",
        priority: "3"
    };

    it("renders without crashing", function () {
        render(<Todo todo={testTodo} />);
    });

    it("matches snapshot", function () {
        const {container} = render(<Todo todo={testTodo} />);
        expect(container).toMatchSnapshot();
    });

});