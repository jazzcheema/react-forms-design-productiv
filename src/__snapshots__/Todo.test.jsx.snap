// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Todo component > matches snapshot 1`] = `
<div>
  <div
    class="Todo"
  >
    <div>
      <b>
        Test Title1
      </b>
       
      <small>
        (priority: 
        3
        )
      </small>
    </div>
    <div>
      <small>
        Test Description1
      </small>
    </div>
  </div>
</div>
`;
